restaurant/src/app/admin/update-status.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@angular/core';
+import { Router } from '@angular/router';
+import { HttpClient,HttpErrorResponse } from '@angular/common/http';
+import { Observable } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UpdateStatusService {
```

Comment:
use appropriate names for service. Eg: UpdateMenuService. if it is updating the menu status, we can use the menu service.  no need for an independent service class for each operation. MenuService can have CRUD operation for the menu.